infrastructure/models.py

```diff
@@ -224,6 +327,134 @@ def get_absolute_url(self):
             }
         )
 
+class Fuel(Publishable):
+    name = models.CharField(max_length=140)
+    fuel_category = models.ForeignKey(
+        'FuelCategory',
+        blank=True, null=True,
+        help_text='Select or create named fuel categories.'
+    )
+    def __str__(self):
+        return self.name
+
+class FuelCategory(Publishable):
+    name = models.CharField(max_length=140)
+
+    class Meta:
+        verbose_name_plural = 'fuel categories'
+
+    def __str__(self):
+        return self.name
+
+
+class PowerPlant(Publishable):
+    """Describes a Power Plant"""
+    name = models.CharField(max_length=140)
+    slug = models.SlugField(max_length=150, allow_unicode=True)
+    infrastructure_type = models.ForeignKey(
+        InfrastructureType,
+        models.SET_NULL, blank=True, null=True,
+        help_text='Select or create named insfrastructure types.'
+    )
+    fuels = models.ManyToManyField(
+        'Fuel', 
+        blank=True, 
+    )
+    countries = models.ManyToManyField('locations.Country', blank=True)
+    regions = models.ManyToManyField(
+        'locations.Region',
+        blank=True,
+        help_text='Select or create geographic region names.'
+    )
+    status = models.PositiveSmallIntegerField(
+        blank=True, null=True,
+        choices=PowerPlantStatus.STATUSES
+    )
+    plant_year_online = models.PositiveSmallIntegerField(blank=True, null=True)
+    plant_month_online = models.PositiveSmallIntegerField(blank=True, null=True)
+    plant_day_online = models.PositiveSmallIntegerField(blank=True, null=True)
+    
+    @property
+    def fuzzy_plant_online_date(self):
+        return fuzzydate(self.plant_year_online, self.plant_month_online, self.plant_day_online)
+    
+    decommissioning_year = models.PositiveSmallIntegerField(blank=True, null=True)
+    decommissioning_month = models.PositiveSmallIntegerField(blank=True, null=True)
+    decommissioning_day = models.PositiveSmallIntegerField(blank=True, null=True)
+
+    @property 
+    def fuzzy_decommissioning_date(self):
+        return fuzzydate(self.decommissioning_year, self.decommissioning_month, self.decommissioning_day)
+
+    plant_capacity = models.FloatField(
+        blank=True, null=True,
+    )
+
+    plant_capacity_unit = models.PositiveSmallIntegerField(
+        blank=True, null=True,
+        choices=ProjectPlantUnits.UNITS
+    )
+
+    plant_output = models.BigIntegerField(
+        blank=True, null=True,
+    )
+
+    plant_output_unit = models.PositiveSmallIntegerField(
+        blank=True, null=True,
+        choices=ProjectPlantUnits.UNITS
+    )
+
+    plant_output_year = models.PositiveSmallIntegerField(blank=True, null=True)
+    estimated_plant_output = models.BigIntegerField(
```

Comment:
It looks like this also can have a decimal in it. We should probably use a `FloatField` here and on `plant_output`